app-libs/sgx-runtime/src/lib.rs

```diff
@@ -256,20 +263,93 @@ impl pallet_parentchain::Config for Runtime {
+
+impl pallet_encointer_scheduler::Config for Runtime {
+	type Event = Event;
+	type OnCeremonyPhaseChange = pallet_encointer_ceremonies::Pallet<Runtime>;
+	type MomentsPerDay = MomentsPerDay;
+	type CeremonyMaster = EnsureAliceOrRoot;
+	type WeightInfo = ();
+}
+
+impl pallet_encointer_communities::Config for Runtime {
+	type Event = Event;
+	type CommunityMaster = EnsureAliceOrRoot;
+	type TrustableForNonDestructiveAction = EnsureSigned<AccountId>;
+	type WeightInfo = ();
+}
+
+impl pallet_encointer_ceremonies::Config for Runtime {
+	type Event = Event;
+	type CeremonyMaster = EnsureAliceOrRoot;
+	type Public = <MultiSignature as Verify>::Signer;
+	type Signature = MultiSignature;
+	// Note: in production networks it is advised to use babes randomness source.
+	// But we have low security requirements here, so it should be fine.
+	type RandomnessSource = pallet_randomness_collective_flip::Pallet<Runtime>;
+	type MeetupSizeTarget = MeetupSizeTarget;
+	type MeetupMinSize = MeetupMinSize;
+	type MeetupNewbieLimitDivider = MeetupNewbieLimitDivider;
+	type WeightInfo = ();
+}
+
+impl pallet_encointer_balances::Config for Runtime {
+	type Event = Event;
+	type DefaultDemurrage = DefaultDemurrage;
+	type ExistentialDeposit = EncointerExistentialDeposit;
+	type WeightInfo = ();
+	type CeremonyMaster = EnsureAliceOrRoot;
+}
+
+impl pallet_asset_tx_payment::Config for Runtime {
+	type Event = Event;
+	type Fungibles = pallet_encointer_balances::Pallet<Runtime>;
+	type OnChargeAssetTransaction = pallet_asset_tx_payment::FungiblesAdapter<
+		encointer_balances_tx_payment::BalanceToCommunityBalance<Runtime>,
+		encointer_balances_tx_payment::BurnCredit,
+	>;
+}
+
 // The plain sgx-runtime without the `evm-pallet`
 #[cfg(not(feature = "evm"))]
 construct_runtime!(
-	pub enum Runtime where
+	pub struct Runtime where
 		Block = Block,
 		NodeBlock = opaque::Block,
-		UncheckedExtrinsic = UncheckedExtrinsic
+		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage},
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>},
+		AssetTxPayment: pallet_asset_tx_payment::{Pallet, Storage, Event<T>},
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
 		Parentchain: pallet_parentchain::{Pallet, Call, Storage},
+		EncointerScheduler: pallet_encointer_scheduler::{Pallet, Call, Storage, Config<T>, Event},
+		EncointerCeremonies: pallet_encointer_ceremonies::{Pallet, Call, Storage, Config<T>, Event<T>},
+		EncointerBalances: pallet_encointer_balances::{Pallet, Call, Storage, Config, Event<T>},
+		EncointerCommunities: pallet_encointer_communities::{Pallet, Call, Storage, Config, Event<T>},
 	}
```

Comment:
Add all encointer pallets because of their dependencies. See https://github.com/encointer/community-sidechain/issues/6

Reply:
Why do you need the indexes?

Reply:
The indexes will define how the aggregated call enum will look like, which is like:

```rust
pub enum RuntimeCall {
   System(SystemCall),
   // Index == defines position in the enum
   EncointerCeremonies(EncointerCeremoniesCall),
   ...
}
```

It is good practice to have this well-defined instead of automatically assigned based on the location where the pallet is declared, and I don't know the exact use case, but it is better if they are the same as on the parachain.

Reply:
Just in case: in integritee-worker there are no indexes

Reply:
not ideal :)